qa/tasks/cephfs/test_mds_metrics.py

```diff
@@ -405,13 +405,13 @@ def test_perf_stats_stale_metrics(self):
         log.debug("metrics={0}".format(metrics))
         self.assertTrue(valid)
 
+        #mount_a and mount_b are the clients mounted for TestMDSMetrics. So get their
+        #entries from the global_metrics.
+        client_a_name = "client.{0}".format(self.mount_a.get_global_id())
+        client_b_name = "client.{0}".format(self.mount_b.get_global_id())
```

Comment:
if possible, please use `f""` string format for formatted stirngs
its a bit more intuitive to understand